src/API/HealthCheck.php

```diff
@@ -44,17 +43,17 @@ public function health_check() {
 
 		try {
 
+			$response = array();
+
 			if ( ! Pinterest_For_Woocommerce()::get_data( 'merchant_id' ) ) {
-				return array(
-					'status' => 'pending_initial_configuration',
-				);
+				$response['status'] = 'pending_initial_configuration';
+				return $response;
```

Comment:
Instead of using the `$response` variable, we can just directly return the array here.

```suggestion
				return array( 'status' => 'pending_initial_configuration' );
```